platform/ios/src/UIViewController+MGLAdditions.h

```diff
@@ -0,0 +1,11 @@
+#import <UIKit/UIKit.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface UIViewController (MGLAdditions)
+
++ (UIViewController *)getTopViewControllerFrom:(UIViewController *)viewController;
```

Comment:
Good point re: `get`. Will fix. This came from the user's suggestion. @jmkiley and I discussed this method name a bit and figured we'd get some more input first.